### PR TITLE
[droid] "Uri too long" in WebView.NavigateToString remediation

### PR DESCRIPTION
**Pull request by pkar70**
GitHub Issue (If applicable): 
#1527

## PR Type

What kind of change does this PR introduce?
- Bugfix

## What is the current behavior?
When WebView.NavigateToString(long_string)
exception is thrown: "uri too long".

## What is the new behavior?
no exception is thrown

## PR Checklist

Please check if your PR fulfills the following requirements:
- [X] Tested code on Android 8.1 Oreo Go (with current [supported SDKs])
- [ ] Docs have been added/updated which fit [documentation template](https://github.com/unoplatform/uno/blob/master/doc/.feature-template.md) (for bug fixes / features)
- [X] [Unit Tests and/or UI Tests](https://github.com/unoplatform/uno/blob/master/doc/articles/uno-development/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)
- [ ] [Wasm UI Tests] (not applicable)
- [X] Contains **NO** breaking changes
- [X] Updated the [Release Notes](https://github.com/unoplatform/uno/tree/master/doc/ReleaseNotes)
- [X] Associated with an issue (GitHub or internal)

### CODE REVIEW

**Comment by pkar70** [comment]
As I don't know how to write automated tests, I can only provide code to manual tests... maybe someone would create test as required by Uno team?

1. XAML:
`       <WebView Name="uiWeb" HorizontalAlignment="Stretch" VerticalAlignment="Stretch" />`

2. C# code:

` {`
`           string sTmp;`
`           sTmp = "<html><body>";`
`           for (int i = 0; i < 1000; i++)`
`           {`
`                string sLinia = "Linia " + i.ToString() + " ";`
`                sLinia = sLinia.PadRight(1000, 'x');`
`                sTmp = sTmp + "<p>" + sLinia;`
`            }`
`            sTmp += "</body></html>";`
`            uiWeb.NavigateToString(sTmp);`
`}`

Without changes from PR, after second or two, exception is thrown. With changes - no exception.

**Comment by jeromelaban** [comment]
A second test is definitely better 👍

File: src/SamplesApp/SamplesApp.UITests/Windows_UI_Xaml_Controls/WebViewTests/UnoSamples_Test_NavigateToString.cs [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+namespace SamplesApp.UITests.Windows_UI_Xaml_Controls.WebView
+{
+	[TestFixture]
+	public partial class WebView_Tests : SampleControlUITestBase
+	{
+		[Test]
+		[AutoRetry]
+		public void WebView_NavigateToLongString()
+		{
+			Run("UITests.Shared.Windows_UI_Xaml_Controls.WebView.WebView_NavigateToString2");
```
Comment (jeromelaban) [inline src/SamplesApp/SamplesApp.UITests/Windows_UI_Xaml_Controls/WebViewTests/UnoSamples_Test_NavigateToString.cs]: This string has to match the `x:Class` name of the xaml test file, this is why your test times out.

Comment (pkar70) [reply src/SamplesApp/SamplesApp.UITests/Windows_UI_Xaml_Controls/WebViewTests/UnoSamples_Test_NavigateToString.cs]: Still Error while performing WaitForElement(Marked("startButton"), "Timed out waiting for element..." , and names are (I think) correct, i.e. same in all three files (xaml, xaml.cs, test.cs)

**Comment by pkar70** [comment]
CI is failing, because test I created to test solution of Android problem - found error in WASM head.

**Comment by pkar70** [comment]
Closed, and recreated as PR#2023.

**Pull request closed without merging**